src/Assets/ScriptWithBuiltDependenciesAsset.php

```diff
@@ -59,7 +59,8 @@ protected function get_dependency_data( string $build_dependency_path, Dependenc
 			if ( ! is_readable( $build_dependency_path ) ) {
 				return $fallback;
 			}
-
+			// Reason of exclusion: These files are being loaded manually in the function call with no user input
+			// nosemgrep audit.php.lang.security.file.inclusion-arg
```

Comment:
According to [the documentation](https://semgrep.dev/docs/ignoring-files-folders-code/) it should always have a colon after `nosemgrep` to specify a specific rule.

```suggestion
			// nosemgrep: audit.php.lang.security.file.inclusion-arg
```

Without the colon it just skips all rules. For example if I add `// nosemgrep foobar` I'd expect it to still detect the warning, and only skip a rule called `foobar` but it skips all checks for that block.